angular/core/components/invitation_form/invitation_form.coffee

```diff
@@ -26,12 +28,25 @@ angular.module('loomioApp').factory 'InvitationForm', ->
     $scope.maxInvitations = ->
       $scope.invitees().length > 100
 
-    $scope.getTeamLink = ->
-      ModalService.open TeamLinkModal, group: -> $scope.form.group()
+    $scope.submitText = ->
+      if $scope.invitees().length > 0
+        'invitation_form.send'
+      else
+        'invitation_form.done'
 
     $scope.canSubmit = ->
       $scope.invitees().length > 0 and $scope.form.group()
 
+    $scope.invitation = ->
+      group.shareableInvitation()
+
+    $scope.copied = ->
+      FlashService.success('common.copied')
+
+    $scope.invitationLink = ->
+      return unless group.shareableInvitation()
+      [AppConfig.baseUrl, 'invitations/', group.shareableInvitation().token].join('')
```

Comment:
I'm a little nervous about hard coding the 'invitations' path into this. Can we instead have the shareable invitation know what url it's supposed to be spitting out?

```
group.shareableInvitation().url
```

```
# invitation_serializer
def url
  invitations_url(group, token) # => or something that spits out http://loomio.org/g/key/invitations/token
end
```
or something like this?